payment/datastore.go

```diff
@@ -115,6 +119,18 @@ func (pg *Postgres) GetOrder(orderID uuid.UUID) (*Order, error) {
 	return &order, nil
 }
 
+// UpdateOrder updates the orders status.
+// 	Status should either be one of pending, paid, fulfilled, or canceled.
```

Comment:
should we add a db constraint for this?

Reply:
https://github.com/brave-intl/bat-go/pull/226/commits/8a8b6e79f2abdb2872e0ec3c00781904da3de72f